src/DssSpell.sol

```diff
@@ -66,45 +60,88 @@ contract DssSpellAction is DssAction, DssSpellCollateralAction {
-
-        // Approve new bridge and cast spell only if the current bridge has closed successfully
-        if(currentBridgeClosed == true && StarknetBridgeLike(currentStarknetDAIBridge).isOpen() == 0){
-            // Bridge code at time of casting: https://github.com/makerdao/starknet-dai-bridge/blob/ad9f53425582c39c29cb3a7420e430ab01a46d4d/contracts/l1/L1DAIBridge.sol
-            address starknetEscrow = DssExecLib.getChangelogAddress("STARKNET_ESCROW");
-            address dai = DssExecLib.getChangelogAddress("MCD_DAI");
-            StarknetEscrowLike(starknetEscrow).approve(dai, NEW_STARKNET_DAI_BRIDGE, type(uint).max);
-            // Relay the L2 spell content
-            // See: https://voyager.online/contract/0x0726cf4161e783ef0043e4998c61ecd70ea3e6f673fd7cf1060130529f849bc2#code
-            address starknetGovRelay = DssExecLib.getChangelogAddress("STARKNET_GOV_RELAY");
-            StarknetGovRelayLike(starknetGovRelay).relay(L2_FEE_SPELL);
-            // ChangeLog
-            DssExecLib.setChangelogAddress("STARKNET_DAI_BRIDGE", NEW_STARKNET_DAI_BRIDGE);
-            DssExecLib.setChangelogAddress("STARKNET_DAI_BRIDGE_LEGACY", currentStarknetDAIBridge);
-        }
-
-        // Bump changelog version either way, due to rETH onboarding
-        DssExecLib.setChangelogVersion("1.14.3");
+        // collateralAction();
+
+
+        // CU Offboarding - Yank DAI Streams
+        // https://forum.makerdao.com/t/executive-vote-cu-offboarding-next-steps/18522
+
+        // Yank DAI Stream #4 (EVENTS-001)
+        // https://mips.makerdao.com/mips/details/MIP39c3SP4#sentence-summary
+        MCD_VEST_DAI.yank(4);
+
+        // Yank DAI Stream #5 (SH-001)
+        // https://mips.makerdao.com/mips/details/MIP39c3SP3#sentence-summary
+        MCD_VEST_DAI.yank(5);
+
+        // Yank DAI Stream #35 (RWF-001)
+        // https://mips.makerdao.com/mips/details/MIP39c3SP5#sentence-summary
+        MCD_VEST_DAI_LEGACY.yank(35);
+
+
+        // CU Offboarding - Yank MKR Stream
+        // Yank MKR Stream #23 (SH-001)
+        // https://mips.makerdao.com/mips/details/MIP39c3SP3#sentence-summary
+        MCD_VEST_MKR_TREASURY.yank(23);
+
+
+        // CU Offboarding - DAI Golden Parachutes
+        // EVENTS-001 - 167,666 DAI - 0x3D274fbAc29C92D2F624483495C0113B44dBE7d2
+        // https://mips.makerdao.com/mips/details/MIP39c3SP4#sentence-summary
+
+        DssExecLib.sendPaymentFromSurplusBuffer(EVENTS_001, 167_666);
+
+        // SH-001 - 43,332.0 DAI - 0xc657aC882Fb2D6CcF521801da39e910F8519508d
+        // https://mips.makerdao.com/mips/details/MIP39c3SP3#sentence-summary
+        DssExecLib.sendPaymentFromSurplusBuffer(SH_001, 43_332);
+
+
+        // CU Offboarding - MKR Golden Parachutes
+        // https://forum.makerdao.com/t/executive-vote-cu-offboarding-next-steps/18522
+
+        // SH-001 - 26.04 MKR - 0xc657aC882Fb2D6CcF521801da39e910F8519508d
+        // https://mips.makerdao.com/mips/details/MIP39c3SP4#sentence-summary
```

Comment:
wrong link, should be https://mips.makerdao.com/mips/details/MIP39c3SP3#sentence-summary ?

Reply:
Also noticed this
```suggestion
        // https://mips.makerdao.com/mips/details/MIP39c3SP3#sentence-summary
```

Reply:
These were pulled off of the governance spreadsheet. Making a note here to check the copy for the correct links.